app/javascript/packages/document-capture/context/acuant.tsx

```diff
@@ -193,8 +193,8 @@ const getActualAcuantCamera = (): AcuantCameraInterface => {
 };
 
 function AcuantContextProvider({
-  sdkSrc = '/acuant/11.7.0/AcuantJavascriptWebSdk.min.js',
```

Comment:
We never followed-up with a pull request to remove [11.5.0](https://github.com/18F/identity-idp/tree/main/public/acuant/11.5.0) after the last upgrade. Can we plan to preemptively get the pull request ready to remove this old version (along with 11.5.0) so it doesn't fall off our radar?